examples/interactivity/add-popups-onhover.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>GeoJSON Layer with CARTO Tiles Example</title>
+
+    <!-- Custom CSS -->
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.css"
+    />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://carto.com/developers/carto-vl/v1.4.4/examples/maps/style.css"
+    />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://libs.cartocdn.com/airship-style/v2.3/airship.min.css"
+    />
+
+    <style>
+      body {
+        margin: 0;
+        padding: 0;
+      }
+
+      #map {
+        width: 100vw;
+        height: 100vh;
+      }
+    </style>
+  </head>
+
+  <body>
+    <aside class="toolbox">
+      <div class="box">
+        <header>
+          <h1>Pop-ups on hover</h1>
+        </header>
+        <section>
+          <p class="description open-sans">Configure pop-up windows on hover</p>
+        </section>
+        <footer class="js-footer"></footer>
+      </div>
+    </aside>
+    <section id="map"></section>
+
+    <!-- Map libraries-->
+    <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.js"></script>
+    <script src="https://unpkg.com/deck.gl@8.1.5/dist.min.js"></script>
+    <!-- Airship -->
+    <script src="https://libs.cartocdn.com/airship-components/v2.3/airship.js"></script>
+
+    <!-- CARTO Deck.gl's Tile Layer -->
+    <script src="/packages/toolkit/dist/umd/index.min.js"></script>
+
+    <script>
+      async function initialize() {
+        carto.setDefaultCredentials({
+          username: 'cartovl',
+          apiKey: 'default_public'
+        });
+
+        const deckMap = carto.viz.createMap({
+          basemap: 'voyager',
+          view: {
+            longitude: 0,
+            latitude: 30,
+            zoom: 1.5
+          }
+        });
+
+        const sizeBinStyle = new carto.viz.sizeBinsStyle('pop_2015', {
+          sizeRange: [1, 50]
+        });
+        const populationLayer = new carto.viz.Layer(
+          'world_population_2015',
+          sizeBinStyle
+        );
+        await populationLayer.setPopupHover([
+          'cartodb_id',
+          { attr: 'pop_2015', title: 'Population' }
+        ]);
+        await populationLayer.addTo(deckMap);
+        window.deckMap = deckMap;
```

Comment:
I would remove the window.deckMap. This is nice for debugging, but we should try to be homogenous on examples. These examples will be a very relevant part of the documentation!